testbench.yaml

```diff
@@ -0,0 +1,5 @@
+migrations:
+  - tests/database/migrations
+
+workbench:
+  install: true
```

Comment:
Having `install: true` and `migrations` other than `false` will migrate the default laravel migrations. Which means we no longer requires `create_password_reset_tokens_table` and `create_users_table`.